Makefile

```
@@ -91,6 +91,7 @@ print-version:
 $(DEEPCOPY_GENERATOR):
 	@echo ">>> Getting deepcopy-gen (for $(DEEPCOPY_GENERATOR))"
 	@$(GO_NOMOD) get k8s.io/code-generator/cmd/deepcopy-gen
+	@$(GO_NOMOD) get -u k8s.io/apimachinery
```

i was tempted to add a new Target, like 

`update apimachinery` or `api k8s` but  i think is ok for now

---

I would not use `-u` (as we do with `deepcopy-gen`)
```suggestion
	@$(GO_NOMOD) get k8s.io/apimachinery
```